### ISSUE TITLE
Quotas don't support size suffix (K/M/G...).

### ISSUE THREAD

**Issue by alexey-milovidov** [ClickHouse/ClickHouse#38426]
```
play-eu :) CREATE QUOTA benchmark
           KEYED BY ip_address
           FOR RANDOMIZED INTERVAL 1 MINUTE MAX query_inserts = 1, written_bytes = '1M',
           FOR RANDOMIZED INTERVAL 1 HOUR MAX query_inserts = 10, written_bytes = '5M',
           FOR RANDOMIZED INTERVAL 1 DAY MAX query_inserts = 50, written_bytes = '20M'
           TO benchmark;

Syntax error: failed at position 116 (''1M'') (line 3, col 73):

CREATE QUOTA benchmark
KEYED BY ip_address
FOR RANDOMIZED INTERVAL 1 MINUTE MAX query_inserts = 1, written_bytes = '1M', <----------------
FOR RANDOMIZED INTERVAL 1 HOUR MAX query_inserts = 10, written_bytes = '5M',
FOR RANDOMIZED INTERVAL 1 DAY MAX query_inserts = 50, written_bytes = '20M'
TO benchmark;

Expected one of: number, end of query

play-eu :) CREATE QUOTA benchmark
           KEYED BY ip_address
           FOR RANDOMIZED INTERVAL 1 MINUTE MAX query_inserts = 1, written_bytes = 100000,
           FOR RANDOMIZED INTERVAL 1 HOUR MAX query_inserts = 10, written_bytes = 500000,
           FOR RANDOMIZED INTERVAL 1 DAY MAX query_inserts = 50, written_bytes = 2000000
           TO benchmark;
```

**Comment by alexey-milovidov** [comment]
@ucasfl maybe someone wants to help?

**Comment by ucasfl** [comment]
> @ucasfl maybe someone wants to help?

I will ask.

**Comment by Ziy1-Tan** [comment]
Can I take this?

**Comment by ucasfl** [comment]
Closed in #49087.
